dg-core/src/test/java/org/finra/datagenerator/engine/scxml/tags/boundary/PositiveBoundHiveDateTest.java

```diff
@@ -62,6 +62,7 @@ public void latestDateTest() {
      * test default behavior
      */
     @Test
+    @Ignore
```

Comment:
@SamerAdra Still want to know why you want to ignore this unit test :) ? @leeny324 is there any known issues with this unit test ?